README.md

```diff
@@ -1 +1,3 @@
-# pull-request-practice
\ No newline at end of file
+# Pull request practice
+
+A repository for practicing pull requests. Request away!
```

Comment:
Is this a detailed enough description?

Reply:
I thought it would be enough. You want more detail? Like what?

Reply:
Like who's doing the practicing here and why.

Reply:
Okay, I'll add more detail. Thanks for the suggestion.

Reply:
Okay I've made some changes